.github/workflows/inductor.yml

```diff
@@ -53,11 +53,11 @@ jobs:
       sync-tag: linux-focal-cuda12_6-py3_10-gcc9-inductor-build
       test-matrix: |
         { include: [
-          { config: "inductor_huggingface", shard: 1, num_shards: 1, runner: "linux.g5.4xlarge.nvidia.gpu" },
-          { config: "inductor_timm", shard: 1, num_shards: 2, runner: "linux.g5.4xlarge.nvidia.gpu" },
-          { config: "inductor_timm", shard: 2, num_shards: 2, runner: "linux.g5.4xlarge.nvidia.gpu" },
-          { config: "inductor_torchbench", shard: 1, num_shards: 2, runner: "linux.g5.4xlarge.nvidia.gpu" },
-          { config: "inductor_torchbench", shard: 2, num_shards: 2, runner: "linux.g5.4xlarge.nvidia.gpu" },
+          { config: "inductor_huggingface", shard: 1, num_shards: 1, runner: "${{ needs.get-label-type.outputs.label-type }}linux.g5.4xlarge.nvidia.gpu" },
```

Comment:
Inductor jobs had deliberately had prefixes removed to keep them always running on Meta runners (since LF was going over budget)

Since we need to opt them into the ephemeral mode though, we have two options:
1. Leave them as is, but keep an eye on LF costs and reduce the % rolled over amount in target determinator accordingly (IMO the better option, but cc @zxiiro in case he has concerns)
2. Edit the get-label-type action to exclude the fleet prefix from it's output.

Reply:
I'm fine adding the prefix. I think it will simplify our reporting too. We'll definitely need to adjust the LF rollout % down once we do this though.